forge-gui/res/cardsfolder/b/balduvian_warlord.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:3 R
 Types:Creature Human Barbarian
 PT:3/2
 A:AB$ RemoveFromCombat | Cost$ T | ActivationPhases$ Declare Blockers | ValidTgts$ Creature.blocking | Defined$ Targeted | UnblockCreaturesBlockedOnlyBy$ Targeted | SubAbility$ ChooseAttacker | SpellDescription$ Remove target blocking creature from combat. Creatures it was blocking that hadn't become blocked by another creature this combat become unblocked.
-SVar:ChooseAttacker:DB$ ChooseCard | Defined$ You | Choices$ Creature.attacking | ChoiceTitle$ Choose an attacker to block | Mandatory$ True | SubAbility$ Block | SpellDescriptions$ None | StackDescription$ None
+SVar:ChooseAttacker:DB$ ChooseCard | Defined$ You | Choices$ Creature.attacking | ChoiceTitle$ Choose an attacker to block | Mandatory$ True | SubAbility$ Block | SpellDescription$ None | StackDescription$ None
```

Comment:
bad change:
![image](https://github.com/Card-Forge/forge/assets/8506892/7629dcd4-d83d-4977-846c-57ab3dd05c60)

Reply:
Oops. Now I recall that `SpellDescriptions$` can be removed without much issue. While I was at it, I added an explicit `TgtPrompt$`.